dash-tx-value-signing.md

```diff
@@ -0,0 +1,250 @@
+<pre>
+DIP: tx-value-signing
+Title: Transaction value signing analogous to BIP143 as implemented in Bitcoin Cash
+Authors: greatwolf, mayoree
+Status: Draft
+Layer: Consensus (hard fork)
+Created: 2021-07-03
+License: MIT License
+</pre>
+
+# Table of Contents
+
+* [Abstract](#abstract)
+* [Motivation](#motivation)
+* [Specification](#specification)
+* [Implementation](#implementation)
+* [Test](#test)
+* [References](#references)
+* [Copyright](#copyright)
+
+# Abstract
+
+This DIP describes a digest algorithm that implements the signature covers value when signing Dash transactions. It opens the path for more efficient signing of Dash transactions on hardware wallets.
```

Comment:
The first sentence is not very clear to me. Looks like it may be part of 2 sentences that got mixed together during editing?